src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -1234,6 +1253,21 @@
       "dataType": "bool",
       "value": "(httpEvaluator && useConfiguration)"
```

Comment:
After https://github.com/unoplatform/uno.templates/pull/1400#discussion_r2053966753 the `httpEvaluator` will no longer be a boolean so the checks involving it will need to evaluate its value (`kiota`,`refit`,`default`,`none`)